pos_order_to_sale_order/static/src/js/CreateOrderButton.esm.js

```diff
@@ -0,0 +1,29 @@
+/** @odoo-module */
+
+import {Component} from "@odoo/owl";
+import {CreateOrderPopup} from "./CreateOrderPopup.esm";
+import {ProductScreen} from "@point_of_sale/app/screens/product_screen/product_screen";
+import {useService} from "@web/core/utils/hooks";
+
+export class CreateOrderButton extends Component {
+    setup() {
+        this.popup = useService("popup");
+    }
+
+    onClick() {
+        this.popup.add(CreateOrderPopup);
+    }
+}
+
+CreateOrderButton.template = "point_of_sale.CreateOrderButton";
```

Comment:
I'm confused, why `"point_of_sale.CreateOrderButton"`, I think we should put `pos_order_to_sale_order` instead of `point_of_sale`, right?